amp.hbs

```diff
@@ -0,0 +1,629 @@
+<!DOCTYPE html>
+<html ⚡>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+
+    <title>{{meta_title}}</title>
+
+    {{amp_ghost_head}}
+
+    <style amp-custom>
```

Comment:
1. Fonts - We have to add all fonts -- [Adding fonts in AMP](https://amp.dev/documentation/guides-and-tutorials/start/add_advanced/fonts/)
![image](https://user-images.githubusercontent.com/36295331/149752068-e6cf627a-477e-4d69-bc78-7178af67a5bc.png)